deep-security-agent/tasks/recommend-security-policy.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+# *********************************************************************
+# * Recommend a security policy for the node
```

Comment:
I think we should use the official term here, ex. "Run recommendation scans" as in 
https://help.deepsecurity.trendmicro.com/10_3/on-premise/Policies/ug-rec-scan.html?Highlight=reco%20scan

Reply:
Rename "Recommend a security policy for the node" to "Run recommendation scans for the node". Also rename this task to run-recommand-scans.

Reply:
Rename corresponding naming in main.yml and README.md.